Testing/Exec/IgnitionDelay/hrTest.sh

```diff
@@ -0,0 +1,25 @@
+# Find first executable name available
```

Comment:
it's unfortunate that we need these script files for these tests. @jrood-nrel do you have an idea for what we could do here? Is this best done within `ctest`?

Reply:
Fixed the header, but I leave this unresolved in case @marchdf wants to try another method